py/desispec/correct_cte.py

```diff
@@ -342,6 +349,31 @@ def get_cte_params(header, cte_params_filename=None):
     return amp_regions, cte_regions
 
 
+def cte_transfer_amount_regnault(pixel, in_trap, alpha, beta,
+                                 cmax, fin, fout):
```

Comment:
Please add a docstring defining the input variables and the functional form for how they are used like you did for the previous `simplified_regnault` function.

Reply:
Updated, resolving.